### ISSUE TITLE
AttributeError with running "python -m jcvi.compara.catalog ortholog -h"

### ISSUE THREAD

**Issue by tanwei123456** [tanghaibao/jcvi#315]
python -m jcvi.compara.catalog ortholog -h

Traceback (most recent call last):
  File "/home/tanwei/miniconda2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/home/tanwei/miniconda2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/jcvi/compara/catalog.py", line 949, in <module>
    main()
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/jcvi/compara/catalog.py", line 80, in main
    p.dispatch(globals())
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/jcvi/apps/base.py", line 111, in dispatch
    globals[action](sys.argv[2:])
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/jcvi/compara/catalog.py", line 658, in ortholog
    p.set_dotplot_opts()
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/jcvi/apps/base.py", line 597, in set_dotplot_opts
    from jcvi.graphics.base import set1
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/jcvi/graphics/base.py", line 395, in <module>
    setup_theme()
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/jcvi/graphics/base.py", line 369, in setup_theme
    import seaborn as sns
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/seaborn/__init__.py", line 9, in <module>
    from .relational import *
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/seaborn/relational.py", line 20, in <module>
    from .axisgrid import FacetGrid, _facet_docs
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/seaborn/axisgrid.py", line 16, in <module>
    from .distributions import distplot, kdeplot,  _freedman_diaconis_bins
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/seaborn/distributions.py", line 16, in <module>
    import statsmodels.nonparametric.api as smnp
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/statsmodels/nonparametric/__init__.py", line 7, in <module>
    from statsmodels.tools._testing import PytestTester
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/statsmodels/tools/__init__.py", line 1, in <module>
    from .tools import add_constant, categorical
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/statsmodels/tools/tools.py", line 8, in <module>
    from statsmodels.compat.python import lzip, lmap
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/statsmodels/compat/__init__.py", line 1, in <module>
    from statsmodels.tools._testing import PytestTester
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/statsmodels/tools/_testing.py", line 11, in <module>
    from statsmodels.compat.pandas import assert_equal
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/statsmodels/compat/pandas.py", line 4, in <module>
    import numpy as np
  File "/home/tanwei/miniconda2/lib/python2.7/site-packages/statsmodels/compat/numpy.py", line 46, in <module>
    NP_LT_114 = LooseVersion(np.__version__) < LooseVersion('1.14')
AttributeError: 'module' object has no attribute '__version__'
    
    I have this problem with Python 2.7 , but I was still blocked by this problem. How can I do to solve this problem?
    Thank you very much!

**Comment by tanghaibao** [comment]
Duplicate of #300

**Comment by tanghaibao** [comment]
@tanwei123456 

Please switch to Python 3 as we are dropping support for Python 2 shortly. If you insist on using Python 2, someone else suggested that you can install a fixed version of numpy in another ticket #300.

Haibao

**Comment by tanwei123456** [comment]
Thank you very much~

**Comment by tanwei123456** [comment]
I try to downgrade statsmodels from 0.11.0 to 0.10.0 (pip install statsmodels==0.10.0), which also solved my problem.
